tests/installation/snapper_rollback_test.pm

```diff
@@ -0,0 +1,67 @@
+# SUSE's openQA tests
+#
+# Copyright © 2009-2013 Bernhard M. Wiedemann
+# Copyright © 2012-2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+
+### the test's goal is to test a rollback (as a backup) after system has migrated.
+### we make one rollback to going back to downgraded system, (1)
+### and we make another rollback to go to upgraded system. at the end 6 snapshots.
+### we are in a read_only_snapsot 1 and after we are in a read_and_write 2
+
+### TOFIX: we could make a cleanup test, to remove the 4 created snapshots.
```

Comment:
use "TODO" for stuff like this, "FIXME" if you want to mark it for yourself, should not appear in commits visible to others